include/drivers/dma.h

```diff
@@ -368,6 +374,57 @@ static inline int z_impl_dma_stop(const struct device *dev, uint32_t channel)
 	return api->stop(dev, channel);
 }
 
+
+/**
+ * @brief Suspend a DMA channel
+ *
+ * Implementations must check the validity of the channel state and ID passed
+ * in and return -EINVAL if it is invalid.
+ *
+ * @param dev     Pointer to the device structure for the driver instance.
+ * @param channel Numeric identification of the channel to suspend
+ *
+ * @retval 0 if successful.
+ * @retval Negative errno code if failure.
```

Comment:
```suggestion
 * @retval 0 If successful.
 * @retval -ENOSYS If not implemented.
 * @retval -errno Other negative errno code if failure.
```

`@retval <value> <description>`

Reply:
Happy to change this as I think you are right, I'll likely change it on all the other functions doc comments then as well since this was copy pasta from the others in this header.